src/dashboard/public/styles.css

```
@@ -1437,12 +1435,53 @@ textarea.auto-resize {
+    content: ""; width: 7px; height: 7px; border-radius: 50%;
+    background: var(--warn); flex-shrink: 0; margin-left: auto;
+}
+.dash-nav .nav-item.has-unsaved.active::after { background: #fff; }
+
+/* The save button that would clear it. */
+.btn.has-unsaved {
+    box-shadow: 0 0 0 2px rgba(245, 158, 11, 0.45);
+}
+.btn.has-unsaved::after {
+    content: ""; display: inline-block; width: 6px; height: 6px;
+    border-radius: 50%; background: currentColor; margin-left: 8px;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)